oras/provider.py

```diff
@@ -348,7 +352,7 @@ def get_blob(
         """
         method = "GET" if not head else "HEAD"
         blob_url = f"{self.prefix}://{container.get_blob_url(digest)}"  # type: ignore
-        return self.do_request(blob_url, method, headers=self.headers, stream=stream)
+        return self.do_request(blob_url, method, headers=self.headers, stream=stream )
```

Comment:
```suggestion
        return self.do_request(blob_url, method, headers=self.headers, stream=stream)
```

Reply:
This (and the trailing comma you edited) will likely fail the formatting check. I will approve this to run but we will still need the DCO signed (let me know if you have questions about that).

Reply:
I signed the commits with my ssh key. Or does it needed with with gpg ? I will fix the formarting.

Reply:
Formarting is back to origin :D

Reply:
It's something called a DCO that you need to sign for CNCF projects - click the link in the failed step below, e.g., https://github.com/oras-project/oras-py/pull/96/checks?check_run_id=15068749650

Reply:
Ahh I didn't know that. Its done